cg/store/models.py

```diff
@@ -511,7 +512,7 @@ class Sample(Model, PriorityMixin):
     sequenced_at = Column(types.DateTime)
     sex = Column(types.Enum(*SEX_OPTIONS), nullable=False)
     subject_id = Column(types.String(128))
-    ticket_number = Column(types.Integer)
+    ticket_number = Column("ticket_number", types.Integer)
```

Comment:
I think we should make ticket number into a constant instead so that we do not hard code
```suggestion
    ticket_number = Column(TICKET_NUMBER, types.Integer)
```